pkg/openservicebroker/server/route.go

```diff
@@ -113,7 +164,12 @@ func provision(b api.Broker, req *restful.Request) *api.Response {
 		return api.NewResponse(http.StatusUnprocessableEntity, &api.AsyncRequired, nil)
 	}
 
-	return b.Provision(instanceID, &preq)
+	u, err := getUser(req)
```

Comment:
for sanity, on provision and bind, is it possible to validate that either X-Broker-API-Originating-Identity is set, or the parameter is set, but not both?